net-analyzer/authforce/authforce-0.9.9-r2.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="An HTTP authentication brute forcer"
+HOMEPAGE="http://www.divineinvasion.net/authforce/"
+SRC_URI="http://www.divineinvasion.net/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="curl nls"
+
+DEPEND="
```

Comment:
I've put ```readline``` and ```curl``` into ```RDEPEND```. I've also added ```gettext``` to BDEPEND. Since ```DEPEND``` is empty now i've added ```DEPEND=${RDEPEND}```, however i'm not sure if that is correct?

Reply:
since presumably both dependencies will be needed at configure/build time that would be correct, yes.